atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -1472,19 +1472,22 @@ private boolean rollbackFailedTransactions(
         }
 
         try {
-            log.warn("For table: " + tableRef
+            log.debug("For table: " + tableRef
                     + " we are deleting values of an uncommitted transaction: " + keysToDelete);
             keyValueService.delete(tableRef, Multimaps.forMap(keysToDelete));
         } catch (RuntimeException e) {
             String msg = "This isn't a bug but it should be infrequent if all nodes of your KV service are running. "
                     + "Delete has stronger consistency semantics than read/write and must talk to all nodes "
                     + "instead of just talking to a quorum of nodes. "
                     + "Failed to delete keys for table" + tableRef
-                    + " from an uncommitted transaction: " + keysToDelete;
-            log.error(msg, e);
+                    + " from an uncommitted transaction; "
+                    + " sweep should eventually clean this when it processes this table.";
+	    if (log.isDebugEnabled()) {
+                msg += " The keys that failed to be deleted during rollback were " + keysToDelete;
```

Comment:
Adv:
  - If lots of transactions are going on at once, they can all become jumbled. Having it appended to the same log message means it is grouped together.
Disadv:
  - It seems super weird to change an `error` level message depending on whether we are logging at the `debug` level.

I think @gbrova is right and we should have it as a debug message.